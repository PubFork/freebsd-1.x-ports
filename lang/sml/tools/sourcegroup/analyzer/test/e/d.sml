
signature A = sig type t val f :unit-> t end

structure Y:A = struct type t = int * int fun f()=(3,4) end

signature B = sig structure Q:A val x:Q.t end

functor F (D:A) :B = struct structure Q:A=D val x:Q.t=Q.f() end

funsig FS (AP:A) = B

functor AA :FS = F

structure AF :B = F (Y)