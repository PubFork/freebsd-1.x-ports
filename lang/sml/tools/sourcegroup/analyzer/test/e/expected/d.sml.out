source sml "e/d.sml"
  export structure AF Y 
  export signature A B 
  export functor AA F 
  export fsig FS ;

